libs/trustchain/src/types.ts

```diff
@@ -65,6 +65,26 @@ export type TrustchainSDKContext = {
   name: string;
 };
 
+/**
+ * the trustchain with a result type indicating what happened during getOrCreateTrustchain
+ */
+export type TrustchainResult =
+  | {
+      // the trustchain didn't exist and was created
+      type: "created";
+      trustchain: Trustchain;
+    }
+  | {
+      // the trustchain already existed and was updated (typically the current member was added)
+      type: "updated";
+      trustchain: Trustchain;
+    }
+  | {
+      // the trustchain existed and was just retrieved (no need to update it)
+      type: "restored";
```

Comment:
Could be possible to use an enum for `type`? 

More easy to use on the App side :)